build.gradle.kts

```diff
@@ -1,7 +1,12 @@
 plugins {
   id("uk.gov.justice.hmpps.gradle-spring-boot") version "3.3.13"
-  kotlin("plugin.spring") version "1.5.31"
-  kotlin("plugin.jpa") version "1.5.31"
+  id("org.unbroken-dome.test-sets") version "4.0.0"
+  kotlin("plugin.spring") version "1.6.0"
```

Comment:
not sure if you meant to do this upgrade since gradle spring boot still targetting 1.5.31?